CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Kiwix JS Contribution Guide
+
+If you have some development experience with HTML and JavaScript, we welcome Pull Requests on existing issues. Please look at this repository's Issue
+tracker, in particular those marked "good first issue". If you have found a bug, check first that we do not have an existing issue covering it, and if not, feel free to create an issue.
+
+Please follow these guidelines when contributing:
+
+- Ask to be assigned to an issue you wish to work on first (we have lots of past issues, some of which are no longer relevant or wanted);
+- We ask you only to pick issues for which you are confident that you have a solution;
+- If you are a newcomer to the repo, create a personal fork and clone it locally. Check out a new branch with a name relevant to the issue you are working on;
```

Comment:
@mossroy Should we change this point to "If you're new to Open Source, please check out these [Contributing Guidelines](https://github.com/kiwix/overview/blob/master/CONTRIBUTING.md) followed by the Kiwix organization."

Reply:
I suggest doing it like this:

- If you are a newcomer to the repo, create a personal fork and clone it locally (for more details, see [Kiwix guidelines](https://github.com/kiwix/overview/blob/master/CONTRIBUTING.md)). Check out a new branch with a name relevant to the issue you are working on;